src/NewBeer.js

```diff
@@ -0,0 +1,63 @@
+import axios from 'axios';
+import React from 'react';
+import {useState} from 'react';
+
+
+export default function NewBeer() {
+const[newBeer, setNewBeer] = useState({
```

Comment:
You could also have a variable for each input field (like [name, setName], [tagline, setTagline], ...) then you don't have to update all object properties when you change a value in one input field